seqr/utils/elasticsearch/es_search.py

```diff
@@ -776,11 +776,19 @@ def _parse_hit(self, raw_hit):
 
         main_transcript_id = sorted_transcripts[0]['transcriptId'] \
             if len(sorted_transcripts) and 'transcriptRank' in sorted_transcripts[0] else None
+        selected_main_transcript_id = None
+        if self._allowed_consequences and sorted_transcripts[0].get('majorConsequence') not in self._allowed_consequences:
```

Comment:
Is sorted_transcripts always guaranteed to have length > 0?
Looking at line 766 it's hard to understand the order of operations with for/in/or 
```
sorted_transcripts = [
            {_to_camel_case(k): v for k, v in transcript.to_dict().items()}
            for transcript in hit[SORTED_TRANSCRIPTS_FIELD_KEY] or []
        ]
```

Will this sorted_transcripts = [[]] sometimes? leading to sorted_transcripts[0] == [] instead of {}?

Reply:
good catch